src/v2/Components/Footer/FooterLink.tsx

```diff
@@ -0,0 +1,10 @@
+import { boxMixin, BoxProps } from "@artsy/palette"
+import styled from "styled-components"
+import { RouterLink, RouterLinkProps } from "v2/Artsy/Router/RouterLink"
+
+export const FooterLink = styled(RouterLink)<RouterLinkProps & BoxProps>`
+  display: flex;
+  text-decoration: none;
+  white-space: nowrap;
+  ${boxMixin}
+`
```

Comment:
We probably didn't need a new file for a single styled component that isn't shared anywhere